idaes/models/unit_models/gibbs_reactor.py

```diff
@@ -212,19 +212,33 @@ def build(self):
 
         # Add performance equations
         # Add Lagrangian multiplier variables
+        # Only need multipliers for species which participate in reactions
+        l_set = []
+        for e in self.control_volume.config.property_package.element_list:
+            skip = True
+            for j in self.control_volume.properties_in.component_list:
+                if j in self.config.inert_species:
+                    continue
+                elif self.control_volume.properties_out.params.element_comp[j][e] != 0:
```

Comment:
AttributeError: '_IndexedGenericStateBlock' object has no attribute 'params' is thrown here. I believe a time index is missing, properties_out[0.0],params should be there instead of properties_out.params

Reply:
This works for me, and I have created a test specifically for your use case to be sure.

Could you check what version of IDAES you are using? I fixed this issue a while ago and StateBlocks now support `.params`.

Reply:
You need to update your version of IDAES as you are at least two full releases behind.

Reply:
This works fine with the latest version. Looks good.